log4j2-elasticsearch-core/src/main/java/org/appenders/log4j2/elasticsearch/RollingIndexNameFormatter.java

```diff
@@ -65,11 +64,12 @@
     private int DEFAULT_BUFFER_SIZE = 32;
     private long currentFileTime;
 
-    protected RollingIndexNameFormatter(String indexName, String pattern, long initTimeInMillis, TimeZone timeZone) {
+    protected RollingIndexNameFormatter(String indexName, String pattern, long initTimeInMillis, TimeZone timeZone, String separator) {
```

Comment:
This change will break extending classes. Could you add a separate constructor with new parameter, please? Current constructor should invoke this with DEFAULT_SEPARATOR.